pkg/forwarder/transaction.go

```diff
@@ -24,6 +24,8 @@ import (
 )
 
 var (
+	connectionDNSSuccess           = expvar.Int{}
```

Comment:
Does it really make sense to track successes? What value do these bring?

Reply:
It tells us that a new DNS lookup was actually made before creating the TCP connection (which makes sense, and always happens with the default `http.Transport`, but I could imagine `http.Transport` implementations trying to cache dns lookup results locally when new creating tcp connections (?)).

In any case, this is very cheap to track (just a counter) so I feel there's no harm tracking them as well.